### PR TITLE
Atreus: add STM32F103C8T6 based variant

### PR DESCRIPTION
**Pull request by DmNosachev**
Handwired Atreus with STM32F103C8T6-based board (Blue Pill or Black Pill)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/atreus/keymaps/quartz64/keymap.c [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+      if (record->event.pressed) {
+        SEND_STRING("O94nx4sUWHc4akud"SS_TAP(X_ENTER));
+      }
+      return false;
+      break;
+	  
+	  case PWD3:
+      if (record->event.pressed) {
+        SEND_STRING("Q123qQ123q"SS_TAP(X_ENTER));
+      }
```
Comment (drashna) [inline keyboards/atreus/keymaps/quartz64/keymap.c:142]: did you mean to leave these in?

Comment (DmNosachev) [reply keyboards/atreus/keymaps/quartz64/keymap.c:142]: I know that using QMK macros to store passwords is a horrible idea from security point of view. But these are passwords for some non-critical services that do not have remote access.

**Review by drashna — APPROVED** [review]

File: keyboards/atreus/keymaps/quartz64/keymap.c [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+	  case PWD2:
+      if (record->event.pressed) {
+        SEND_STRING("O94nx4sUWHc4akud"SS_TAP(X_ENTER));
+      }
+      return false;
+      break;
+	  
+	  case PWD3:
+      if (record->event.pressed) {
+        SEND_STRING("Q123qQ123q"SS_TAP(X_ENTER));
```
Comment (fauxpark) [inline keyboards/atreus/keymaps/quartz64/keymap.c:141]: You can do this and save some bytes:
```suggestion
        SEND_STRING("Q123qQ123q\n");
```

File: keyboards/atreus/keymaps/quartz64/keymap.c [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+	  
+	  case MC2:
+      // NB-space + mdash + space
+      if (record->event.pressed) {
+        SEND_STRING(SS_TAP(X_RALT)"  "SS_TAP(X_RALT)"--- ");
+      }
+      return false;
+      break;
+	  
+	  case MC3:
```
Comment (fauxpark) [inline keyboards/atreus/keymaps/quartz64/keymap.c:117]: Please be mindful of trailing whitespace, and mixing tabs & spaces.
```suggestion
	  case MC3:
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/atreus/keymaps/quartz64/keymap.c [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+	  case PWD1:
+      if (record->event.pressed) {
+        SEND_STRING("NakedLunch1991"SS_TAP(X_ENTER));
+      }
+      return false;
+      break;
+	  
+	  case PWD2:
+      if (record->event.pressed) {
+        SEND_STRING("O94nx4sUWHc4akud"SS_TAP(X_ENTER));
```
Comment (fauxpark) [inline keyboards/atreus/keymaps/quartz64/keymap.c:134]: etc.

Comment (DmNosachev) [reply keyboards/atreus/keymaps/quartz64/keymap.c]: Fixed according to coding conventions and documentation

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**